### PR TITLE
Corresponding to issues 2, 3, 4, 5, 6

### PR DESCRIPTION
**Pull request by Medha-B**
Regarding issues encountered with branch_cui.

### CODE REVIEW

**Comment by kaitoii11** [comment]
This is really hard to compare. Can you change the base branch to branch_cui so we'll easily know the diff?

And one tip. Keep the commit small so anytime you have to revert changes, the size of change will be small.
For example, have 5 commits each for the issue instead of a big 1 commit.

**Comment by Medha-B** [comment]
Will there be some kind of issue if I changed the base now? Also, how to change the base? 
Note: I had messed up a little with branch_cui....it shows "rejected: non-fast-forward" when I try to push anything.

**Comment by kaitoii11** [comment]
There shouldn't be any issue just by changing the base. It just specifies to what branch you want this branch to be merged to.

> Note: I had messed up a little with branch_cui....it shows "rejected: non-fast-forward" when I try to push anything.

Maybe try doing `git pull`  in branch_cui

**Comment by Medha-B** [comment]
I did "git pull" for branch_cui. But it said nothing to fetch, everything up to date. So, I cannot figure out how to fix it.

**Comment by Medha-B** [comment]
Even so, would you prefer if changed the branch to branch_cui?

**Comment by Medha-B** [comment]
Even so, would you prefer it if I changed the branch to branch_cui?

**Comment by kaitoii11** [comment]
> I did "git pull" for branch_cui. But it said nothing to fetch, everything up to date. So, I cannot figure out how to fix it.

Let's see if we can solve it during the meeting

> Even so, would you prefer it if I changed the branch to branch_cui?

Yes, since all the changes in branch_cui and xitosbml_cui are in this branch,  I can't tell what has actually changed in this branch.

File: src/main/java/jp/ac/keio/bio/fun/xitosbml/cui/CuiRun.java [hunk 1]
```diff
@@ -48,5 +49,5 @@ public static void main(String... args) {
  * @SuppressWarnings("deprecation") public static void main(String... args) {
  * CommandLine.run(new CuiRun(), System.err, args); }
  * 
- * }
- */
\ No newline at end of file
+ * } <<<<<<< HEAD
```
Comment (kaitoii11) [inline src/main/java/jp/ac/keio/bio/fun/xitosbml/cui/CuiRun.java]: You should delete `<<<HEAD` part

Comment (Medha-B) [reply src/main/java/jp/ac/keio/bio/fun/xitosbml/cui/CuiRun.java]: Sorry! I had not been careful. Removed <<<Head part.

**Comment by kaitoii11** [comment]
LGTM

**Comment by Medha-B** [comment]
So, should I merge this pull request? Can I also close the issues 2, 3, 4, 5, 6 then? Also, when this branch is merged with branch_cui, would I also merge branch_cui with the master?

**Comment by kaitoii11** [comment]
Yes you should merge. And close the issues.
After merging this pull request, you should merge branch_cui with master

**Pull request merged**